src/pages/ExperimentsAdmin.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Grid, Row, Col, Button } from 'react-bootstrap';
+import { Helmet } from 'react-helmet';
+import PageHeader from '../components/PageHeader';
+import Loader from '../components/Loader';
+import ExperimentsAdminList from '../components/ExperimentsAdminList';
+import {
+  load as experimentsLoad,
+  create as experimentCreate,
+} from '../state/experiments';
+
+class ExperimentsAdmin extends Component {
+  componentDidMount() {
+    this.props.load();
```

Comment:
no need to fix it, just to let you know. Here we have kind of "bug" here. Because user will come here from another page in most of the cases we will have experiments in the store already. So a user will see the list of experiments and in few milliseconds it will be replaced by a loader and then by list again.

Reply:
This was actually copied from Experiments.js.
I guess one way to handle this would be to have an expiration date for the data, and then the function could load only if the list is empty, or the expiration date passed... But for now in my opinion a forced load is not that inconvenient for the user experience.